src/dmd/statementsem.d

```diff
@@ -3712,17 +3712,20 @@ else
             v.dsymbolSemantic(sc);
             cs.push(new ExpStatement(ss.loc, v));
 
-            auto args = new Parameters();
-            args.push(new Parameter(0, t.pointerTo(), null, null, null));
+            auto enterArgs = new Parameters();
+            enterArgs.push(new Parameter(0, t.pointerTo(), null, null, null));
```

Comment:
This seems to be just a refactoring. Is this really necessary to backport to stable?

Reply:
Please see the commit msg, it's a fix.

Reply:
I am not talking about the entire commit, but for this specific renaming from `args` to `enterArgs`. Maybe I am missing something obvious, but to me it just looks like a name change. If we leave the lines 3715 and 3716 untouched, will that invalidate the fix?

Reply:
It *is* a name change, emphasizing that it's the param types for the enter-function only, not the exit function too as previously. Sorry but I don't think debating over this is worth our time. ;)